### ISSUE TITLE
xrdp in kali linux not functioning on install.

### ISSUE THREAD

**Issue by lewkal** [neutrinolabs/xrdp#1998]
PLEASE bear with me, i am new to github.....

RE: error: Failed to start xrdp.service: Unit xrdp.service not found. 

this is the error i am facing when running the following command(s) 
systemctl start xrdp 

for some reason this error will not go after trying to fix and diagnose via the following commands:

sudo ufw disable [disable firewall]

systemctl enable xrdp 
 
sudo dpkg -l "*xrdp*" 

systemctl unmask xrdp

i have gotten down to everything including 'unmasking the service' but still a no go i get hit with the following error...
['Failed to start xrdp.service: Unit xrdp.service not found']  

i was told this was maybe a 'corrupt' package database error so i have tried reconfiguring with:  sudo  dpkg ––configure –a 
thinking maybe it is unpacked but not installed but nothing there either, no output

can anyone help??

**Comment by matt335672** [comment]
Have you installed the `xrdp` and `xorgxrdp` packages?

What does this command tell you?

```
dpkg -L xrdp | grep xrdp.service
```

**Comment by lewkal** [comment]
its seems as though xrdp or any packages i try to install from the repository just do not install if that makes sense, i seen a similar issue i think a year ago on this subject and someone suggested something to do with pkg database i run the following command you provided and was given with this output:

 dpkg -L xrdp | grep xrdp.service
dpkg-query: package 'xrdp' is not installed
Use dpkg --contents (= dpkg-deb --contents) to list archive files contents.

basically nothing from repos will install... i am using a VM if this helps..

**Comment by matt335672** [comment]
I can't really help you with that. I know Kali is based on Debian, but I've never installed it.

Check your `/etc/apt/sources.list` is correct - see [this page](https://www.kali.org/docs/general-use/kali-linux-sources-list-repositories/). If you've installed from offline media, it seems you may not have anything useful configured here.

I'm closing this issue, as it's nothing to do with xrdp.
